### PR TITLE
chore: update link to developer documentation install page

### PR DESCRIPTION
**Pull request by awentzel**
# Description

Update to link into start of developer documentation.  Currently, this start page/getting started is work in progress.

## Motivation & context

When clicking on current link it takes you to the developer documentation main readme.  This is great if that readme contains getting started or different documentation. But, on initial load of this page it's the same exact content and this might be confusing to users.  They may immediately click away without seeing out other developer documentation.  This will temporarily change the link to the install page of documentation.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://microsoft.github.io/fast-dna/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: docs/en/contributing/packages.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ These may include:
 
 All packages use npm and are published to the [npm registry](https://www.npmjs.com/).
 
-### "sideEffects": false
+### Side effects
```
Comment (awentzel) [inline docs/en/contributing/packages.md]: The navigation menu look cluttered as a result of code in a title.  This cleans that up and adds clarity in the first sentence.

File: docs/en/contributing/working.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ The above outlines the format for the **final** commit, but contributors have ma
 
 Typical interim commits may look like the following:
 
-```terminal
+```bash
```
Comment (awentzel) [inline docs/en/contributing/working.md]: this change helps the rendered markdown look cleaner

File: docs/en/contributing/install.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ Clone the repository, `cd` into the project, install [Lerna](https://github.com/
 ```shell
 git clone https://github.com/Microsoft/fast-dna.git
 cd fast-dna
-npm i --global lerna@3.3.0
+npm i --global lerna
```
Comment (awentzel) [inline docs/en/contributing/install.md]: the correct version is installed on `npm i` and it's not necessary to explicitly call out the version that will frequently change over time and could grow out of sync.

Comment (chrisdholt) [reply docs/en/contributing/install.md]: Can we stop from pinning during the install in the build process then as well?

Comment (chrisdholt) [reply docs/en/contributing/install.md]: Can we update the build server as well then?

Comment (awentzel) [reply docs/en/contributing/install.md]: Yep.

**Pull request merged**